src/UserInterface/Defaults/AppDefaults.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2024 Paweł Gilewicz
+ *
+ * This file is part of the Mesh Generating Tool. (https://github.com/PawelekPro/MeshGeneratingTool)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AppDefaults.h"
+
+const QString AppDefaults::_templatesPath = ":/templates/templates/ProjectSetup.json";
+const QString AppDefaults::_comboBoxModelsPath = ":/templates/templates/ComboboxModels.json";
+
+//--------------------------------------------------------------------------------------
+AppDefaults::AppDefaults() {
```

Comment:
To work properly on other platforms a format for settings file has to be defined. This can be done easily by just using the initializer list for AppDefaulst. example: AppDefaults::AppDefaults() : settings("settingsFile.ini", QSettings::IniFormat). Please check if that works on Your platform or if another format is required.

Reply:
Done. Looks like it works on Ubuntu

![image](https://github.com/PawelekPro/MeshGeneratingTool/assets/107176424/f75e8046-7184-4744-9067-aa52ba2ae170)